### PR TITLE
feat(Switch): consumed Penta tokens

### PR DESCRIPTION
**Pull request by thatblindgeye**
Work towards #5738

### CODE REVIEW

**Comment by andrew-ronaldson** [comment]
Screenshot 2023-12-16 at 8 07 29 AM
One nit.
The dot in the switch seems to be aligned to the bottom. Can it be vertically centered?

**Comment by thatblindgeye** [comment]
@andrew-ronaldson can you take another look at the alignment?

**Review by andrew-ronaldson — APPROVED** [review]
Thanks for the update. LGTM

**Review by srambach — APPROVED** [review]
LGTM! 🚀

File: src/patternfly/components/Switch/switch.scss [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
+  --#{$switch}__input--focus__toggle--OutlineOffset: var(--pf-t--global--spacer--xs);
+  --#{$switch}__input--focus__toggle--OutlineColor: var(--pf-t--global--color--brand--default);
 
   // Switch toggle
   --#{$switch}__toggle--Height: calc(var(--#{$switch}--FontSize) * var(--#{$switch}--LineHeight));
-  --#{$switch}__toggle--BackgroundColor: var(--#{$pf-global}--palette--black-500);
-  --#{$switch}__toggle--BorderRadius: var(--#{$switch}__toggle--Height);
+  --#{$switch}__toggle--BackgroundColor: var(--pf-t--global--background--color--control--default);
+  --#{$switch}__toggle--BorderColor: var(--pf-t--global--border--color--default);
+  --#{$switch}__toggle--BorderWidth: var(--pf-t--global--border--width--regular);
```
[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/components/Switch/switch.scss:38]: Also in the design there is no border when the switch is checked. Can we remove that when it's checked (or add it when it's not checked)? It also means we probably need to move the border to the `::after` psudo element so we can add/remove it without changing the shape of the __toggle element. 

IMO the border should be removed from the disabled switch, too - it's so close to the disabled background color that you can't really tell it's a border. To me it makes the edge of the switch look a little fuzzy like a small box-shadow. But that's more a question for @lboehling and @andrew-ronaldson

**Review by mcoker — APPROVED** [review]

**Pull request merged**